src/resolve/resolved-dns-query.c

```diff
@@ -480,6 +480,14 @@ int dns_query_new(
 
         assert(m);
 
+        /* Check for records that is refused and refuse query for the records if matched in configuration */
+        DNS_QUESTION_FOREACH(key, question_utf8) {
```

Comment:
Please drop unnecessary `{}`.

Reply:
@yuwata Yeah, misread this code, it was one block so yeah, i've removed the curly braces.